examples/kafka-consumer-group-service/kafka_consumer_group_service.description

```diff
@@ -1,4 +1,4 @@
-// Here, is an example of a kafka consumer used as a listener to listen to a
+// Here is an example of a kafka consumer used as a listener to listen to a
```

Comment:
```suggestion
// This is an example of a Kafka consumer used as a listener to listen to a
```
Need to make the first letter of `Kafka` uppercase. Please check all occurrences.